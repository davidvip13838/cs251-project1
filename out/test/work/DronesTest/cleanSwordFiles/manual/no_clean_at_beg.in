5 5 10
10
10
10
10
10
0
0
3
10
22
