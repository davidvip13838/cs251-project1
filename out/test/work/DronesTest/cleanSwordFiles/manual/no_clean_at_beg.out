10 10
20 20
30 27
40 30
50 28
